contexts/credentials/v2

```diff
@@ -92,6 +92,28 @@
       }
     },
 
+    "JsonSchema2023": {
+      "@id":
+        "https://www.w3.org/ns/credentials#JsonSchema2023",
```

Comment:
```suggestion
        "https://w3.org/2018/credentials#JsonSchema2023",
```

For now, it'll have to be this because `https://www.w3.org/ns/credentials#` doesn't resolve. We can raise an issue to contemplate splitting the vocabulary URLs between ""https://w3.org/2018/credentials#" and "https://www.w3.org/ns/credentials#". I expect the discussion to be contentious because there are good reasons for and against such a change. If we want to establish two namespaces for credentials, we'll have to have that discussion, but that's no reason to hold up this PR.

If we just accept the changes above (and raise an issue to talk about this), then we can merge the PR. If we don't change the URL above, then this PR will be blocked on that discussion.